spec/features/catalog_view_spec.rb

```diff
@@ -126,15 +127,17 @@
       expect(page.body).to include "Missionaries"
       expect(page.body).to include "Genre Terms"
       expect(page.body).to include "Correspondence"
-      expect(page.body).to include "Names"
-      expect(page.body).to include "Foreign Service Institute"
+      # Commented out these two - names are not currently in our aspace
+      # instance data.
+      # expect(page.body).to include "Names"
+      # expect(page.body).to include "Foreign Service Institute"
```

Comment:
suggestion: maybe this could be a `TODO:` so we know to go back and check when names eventually make it into aspace.